core/pom.xml

```diff
@@ -69,6 +69,12 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why put postgres's dependencies in the core module instead of the rm-datasource module？